server/heroku/processes.go

```diff
@@ -91,6 +92,15 @@ func (h *PostProcess) ServeHTTPContext(ctx context.Context, w http.ResponseWrite
 
 		fmt.Fprintf(outStream, "HTTP/1.1 200 OK\r\nContent-Type: application/vnd.empire.raw-stream\r\n\r\n")
 
+		// We send the null character periodically, to keep the connection alive.
+		// Otherwise the connection would close after the ELB idle connection timeout.
+		go func() {
+			for {
+				fmt.Fprintf(outStream, "\x00")
```

Comment:
We only really need to do this every 10-30 seconds. I think we'll also want to make sure that this goroutine stops when the request has stopped. We can select on a couple channels like this:

```go
stop := make(chan struct{})
go heartbeat(w)

// Do stuff

close(stop)

func heartbeat(stop chan struct{}, w io.Writer) {
        // Send heartbeats every 10 seconds.
        t := time.NewTicker(10*time.Second)

        go func() {
                for {
                        select {
                        case <-t: // Tick event
                                fmt.Fprintf(outStream, "\x00")
                                continue
                        case <-stop:
                                break
                        }
                        break
                }
        }()
}
```